alpha/lib/model/entry.php

```diff
@@ -908,7 +912,38 @@ public function getArchiveExtension()
 	{
 		return $this->archive_extension;
 	}
-	
+
+	/**
+	 * The method returns the var_dumpd_options, in case the entry was created via clone operation.
+	 * Otherwise; it returns null
+	 * @return null
+     */
+	public function getClonedOptions()
+	{
+		return $this->cloned_options;
+	}
+
+
+	/**
+	 * The method sets the options by which the current entry was cloned.
+	 * In case the current entry was not cloned,the $cloneOptionsArray is empty
+	 * @param array $cloneOptionsArray
+     */
+	public function setClonedOption(array $cloneOptionsArray)
```

Comment:
The following code will do the same.
public function setClonedOption(array $cloneOptionsArray=array())
{
    $this->cloned_options = $cloneOptionsArray
}